### PR TITLE
Zenas

### PR DESCRIPTION
**Pull request by Rume7**
Kindly review this Python file.

### CODE REVIEW

**Review by zenas-awuku — APPROVED** [review]

**Comment by Rose-AM** [comment]
Done

File: coder.py
Comment (Rose-AM) [inline coder.py:1]: This file is empty. Kindly correct

**Review by Rose-AM — CHANGES_REQUESTED** [review]

File: coder.py
Comment (zenas-awuku) [inline coder.py:1]: This file is empty

**Review by zenas-awuku — CHANGES_REQUESTED** [review]

**Comment by Rume7** [comment]
I have added the read csv line.

**Pull request closed without merging**